src/Microsoft.IdentityModel.Tokens/CryptoProviderFactory.cs

```diff
@@ -149,8 +149,9 @@ private bool IsSupportedKeyWrapAlgorithm(string algorithm, SecurityKey key)
         /// </summary>
         /// <param name="algorithm">the algorithm to check.</param>
         /// <param name="key">the <see cref="SecurityKey"/>.</param>
+        /// <param name="isPrivateKey">Whether the <see cref="SecurityKey"/> is private key then set this to true.</param>
         /// <returns>true if 'algorithm, key' pair is supported.</returns>
-        public virtual bool IsSupportedAlgorithm(string algorithm, SecurityKey key)
+        public virtual bool IsSupportedAlgorithm(string algorithm, SecurityKey key, bool isPrivateKey)
```

Comment:
Yes, you are right. This is a breaking change. I neglect "Public" keyword by mistake...